shopinvader/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1 @@
+This is shopinvader the odoo module for the new generation of e-commerce
```

Comment:
What about a longer description here?

Maybe add the main pitch from the shopinvader website ?

> ShopInvader is an ecommerce software to create and manage easily your online store with Odoo.

Reply:
I think a link should be interesting also

> This is the Odoo side of [Shopinvader](https://shopinvader.com/)